source/installation/install-with-docker.txt

```diff
@@ -0,0 +1,142 @@
+.. _rm-install-docker:
+
+===================
+Install with Docker
+===================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install and run Relational Migrator as a Docker container. 
+Running Relational Migrator with Docker ensures an installation process 
+that is both independent of the operating system and straightforward 
+to replicate.
+
+About this Task
+---------------
+
+You can also use `Docker Compose <https://docs.docker.com/compose/>`__ 
+to run Relational Migrator. You can find a Docker Compose example file
+in the 
+`download center <https://www.mongodb.com/try/download/relational-migrator>`__.
+
+.. #TODO# Update the link, when the compose file is added. 
```

Comment:
This link will be fine.

Reply:
Ok removing this comment, remaining for the original download link